async/async-and-await/vb/AsyncFirstExampleVB/AsyncFirstExampleVB.sln

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Since this is a one-project solution, the solution file can be removed. All project files can be placed in the async/async-and-await/vb directory.

Reply:
All suggested edit are committed. Solution file is removed. All project files are now in the async/async-and-await/vb directory.